elyra/pipeline/pipeline_definition.py

```diff
@@ -625,9 +616,14 @@ def propagate_pipeline_default_properties(self):
                     merged_list = ElyraPropertyList.merge(node_value, pipeline_value)
                     node.set_component_parameter(property_name, merged_list)
 
+            # Runtime image is a special case that must be handled separately
```

Comment:
Why is this a special case?  Can we extend this comment to clarify for future maintainers?

I noticed above that it doesn't map to an `ElyraProperty` class hierarchy and wondering if that would simplify its handling in these kinds of places.

Reply:
Yeah runtime image is sort of a tricky case. Like you say, it's the only pipeline default property (right now) that doesn't have a corresponding `ElyraProperty`. I played around with making it an `ElyraProperty` subclass, but there's so much special processing of runtime image in the processors and validation that it became pretty messy. 

I think what I might do instead of this special handling block is to define a new property on `Node` that corresponds to the set of properties that _should_ be propagated. Then the docstring for that property can explain why in some cases `elyra_owned_properties != <the set of properties to propagate>`.